tests/foreman/cli/test_user.py

```diff
@@ -1091,6 +1093,45 @@ def test_positive_remove_role(self):
                 user = User.info({'id': user['id']})
                 self.assertNotIn(role_name, user['roles'])
 
+    @pytest.mark.skip_if_open("BZ:1765052")
+    @tier2
+    def test_positive_last_login_for_new_user(self):
+        """Create new user with admin role and check last login updated for that user
+
+        :id: 967282d3-92d0-42ce-9ef3-e542d2883408
+
+        :expectedresults: last login should be updated for user after login using hammer
+
+        :BZ: 1765052
+
+        :CaseLevel: Integration
+        """
+        login = gen_string('alpha')
+        password = gen_string('alpha')
+        org_name = gen_string('alpha')
+        make_user({'login': login, 'password': password})
+        User.add_role({'login': login, 'role': 'System admin'})
+        result_before_login = User.list()
```

Comment:
Why list all users and then cycle through all of them? How about only selecting the user we need? Something like:
```suggestion
        result_before_login = User.list({
                    u'search': u'login = {0}'.format(login),
        })
```

Reply:
somehow I was not able to recall this, thanks for pointing out